app/controllers/usages_controller.rb

```diff
@@ -0,0 +1,46 @@
+class UsagesController < ApplicationController
+  before_action :set_usage, only: [:edit, :update]
+  before_action :initialize_usage, only: [:new, :create]
+  before_action :set_subscription, only: [:new, :edit, :create, :index]
+  before_action :set_subscriptions_features, only: [:new, :edit]
+
+  def create
+    if @subscription.create_usage(usage_params)
```

Comment:
does this syntax work?
Shouldn't it be `usage.create`?

Reply:
`create_usage` is a model method of Subscription so we do not have to pass `subscription_id` in params

Reply:
remove the model method and do as advised above.